cd/cd.yml

```diff
@@ -526,6 +525,7 @@ Resources:
                 Branch: master
                 OAuthToken: !Ref GitHubToken
                 PollForSourceChanges: false
+              Namespace: infrastructure_repo
```

Comment:
Ah, okay, so if you don't specify a Namespace on an ActionDeclaration ... you can't lookup variables produced by it later on.

Reply:
Yeah, not really clear why they didn't just reuse the action's `Name`, since they had pretty much the same uniqueness requirements. But yeah, adding a NS makes the variables active.